### ISSUE TITLE
Reservation with payment not picked up after payment went through

### ISSUE THREAD

**Issue by grimpy** [threefoldtech/tfexplorer#21]
https://explorer.devnet.grid.tf/explorer/reservations/124
Reservation got canceled while waiting for payment

![photo_2020-04-27_10-28-42](https://user-images.githubusercontent.com/143869/80351008-e7cee780-8871-11ea-8911-464c2d14a8c4.jpg)

**Comment by grimpy** [comment]
```
8f28c271521e5b9b08b5a15d0dd83c129089c964507be048d391b79b5543c5ac created at 2020-04-27T08:03:06Z with memo text '124', 890330350e647407c7f7b32f76f6d37024d3e033c25af652c32e1dd94f7b2edc created at 2020-04-27T08:19:34Z with memo text '124', 
```

**Comment by LeeSmet** [comment]
From the logs on the node:

**Comment by LeeSmet** [comment]
Fixed in 4158b45b49a5afe3a3b4c37cdb33557e01c4b244
